tests/test_unsupported.py

```diff
@@ -270,6 +277,39 @@ def test_snippet(self, case, lang):
         if not expect_success:
             assert False
 
+    # These tests are expected to fail for all languages
+    @foreach(sorted(TEST_ERROR_CASES.keys()))
+    def test_error_cases(self, case):
+        env = os.environ.copy()
+        lang = "rust"  # Just so we avoid running tests N times
+        if ENV.get(lang):
+            env.update(ENV.get(lang))
+
+        settings = _get_all_settings(Mock(indent=4, extension=False), env=env)[lang]
+        ext = settings.ext
+        source_data = case
+        expected_error = TEST_ERROR_CASES[case]
+        is_script = has_main(source_data)
+        if ext in [".dart", ".kt", ".rs"] and not is_script:
+            source_data = f"def main():\n  {source_data}"
+        print(f">{source_data}<")
+        tree = ast.parse(source_data)
+        astpretty.pprint(tree)
+        proc = run([sys.executable, "-c", source_data], capture_output=True)
```

Comment:
this is not exercising the code to prove it is usable Python code.
and these cases are not usable.
```py
python -c "a: i8 = 10; b: i16 = 300; c: i16 = a + b"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'i8' is not defined
```